src/etl/Source.cpp

```diff
@@ -59,7 +62,12 @@ make_Source(
         std::move(onConnect),
         std::move(onDisconnect)
     );
-    impl::ForwardingSource forwardingSource{ip, wsPort};
+
+    auto const cacheEntryTimeout = config.valueOr<uint64_t>("cache_entry_timeout", 0);
```

Comment:
Was this meant to be `cache_timeout_ms`? Or the example config is wrong - must be one of the two i guess :)